app/src/main/java/com/hedvig/app/util/featureflags/RemoteFeatureFlagProvider.kt

```diff
@@ -0,0 +1,46 @@
+package com.hedvig.app.util.featureflags
+
+import com.hedvig.app.feature.settings.Market
+import com.hedvig.app.feature.settings.MarketManager
+import com.hedvig.app.service.RemoteConfig
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+
+class RemoteFeatureFlagProvider(
+    private val marketManager: MarketManager,
+    private val remoteConfig: RemoteConfig,
+) : FeatureFlagProvider {
+
+    override val priority = REMOTE_PRIORITY
+
+    private var seCampaignVisible: Boolean = false
+    private var noCampaignVisible: Boolean = false
+    private var dkCampaignVisible: Boolean = false
+
+    init {
+        CoroutineScope(Dispatchers.IO).launch {
+            seCampaignVisible = remoteConfig.fetch().seCampaignVisible
```

Comment:
Does `RemoteConfig::fetch` carry a penalty on a per-call basis? If yes, this could be a viable option:
```suggestion
            val remoteConfigData = remoteConfig.fetch()
            seCampaignVisible = remoteConfigData.seCampaignVisible
```

If not, disregard this comment, unless you feel that it improves readability.

Reply:
Good catch, will fix